docs/pages/modules/native-view-tutorial.mdx

```diff
@@ -108,7 +108,7 @@ Now let's run the example project to make sure everything is working. We'll need
   ]}
 />
 
-We should now see... a blank purple screen. That's not very exciting, but it's a good start. Let's actually make it a WebView now.
+We should now see a blank purple screen. That's not very exciting, but it's a good start. Let's actually make it a WebView now.
```

Comment:
```suggestion
We should now see a blank purple screen. That's not very exciting. However, it's a good start. Let's make it a WebView now.
```

Update to not use words like but and also remove unnecessary words like "actually".